webxdc.js

```diff
@@ -1,125 +1,82 @@
 // debug friend: document.writeln(JSON.stringify(value));
-window.xdcStorage = (() => {
+window.webxdc = (() => {
+    var updateListener = () => {};
     var updatesKey = "__xdcUpdatesKey__";
-    var fakeStorage = {
-        _data: {},
-
-        setItem: function (id, val) {
-            return this._data[id] = String(val);
-        },
-
-        getItem: function (id) {
-            return this._data.hasOwnProperty(id) ? this._data[id] : undefined;
-        },
-
-        removeItem: function (id) {
-            return delete this._data[id];
-        },
-
-        clear: function () {
-            return this._data = {};
-        }
-    };
-    var localStorageSupported = () => {
-        var testKey = "__xdcTestKey__";
-        try {
-            var storage = window.localStorage;
-            storage.setItem(testKey, "1");
-            storage.removeItem(testKey);
-            return true;
-        } catch (error) {
-            return false;
+    window.addEventListener('storage', (event) => {
+        if (event.key == null) {
+            window.location.reload();
+        } else if (event.key == updatesKey) {
```

Comment:
```suggestion
        } else if (event.key === updatesKey) {
```

iirc, in general, it is advisable in javascript to use the "strict equality" operator, that are three equal signs.